frontend/src/metabase/metadata/pages/DataModel/components/SectionPill/SectionPill.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box, Flex, Icon, type IconName, Text } from "metabase/ui";
+
+import S from "./SectionPill.module.css";
+
+interface Props {
+  icon: IconName;
+  title: string;
+}
+
+export const SectionPill = ({ icon, title }: Props) => {
```

Comment:
Mantine offers a Pill component that we could probably use here instead. It's `display: inline-flex` so it might save some boilerplate.

Reply:
- > Pill component is designed to be used inside inputs. It can be used to create custom multi select or tag inputs.
- we don't have this component in `metabase/ui` yet, and avoiding passing one prop to `Flex` does not really justify adding it to our library